src/app/shared/components/map/map.component.ts

```diff
@@ -34,8 +37,7 @@ export class MapComponent implements AfterViewInit, OnDestroy {
   @Output() setAddressEvent = new EventEmitter<Address>();
   @Output() selectedAddress = new EventEmitter<Address>();
 
-  constructor(private geolocationService: GeolocationService) { }
-
+  constructor(private geolocationService: GeolocationService,private previousUrl: PreviousUrlService) { }
```

Comment:
spacing before 'private'
rename please previousUrl to previousUrlService
`  constructor(private geolocationService: GeolocationService, private previousUrl: PreviousUrlService) { }
`